bot/processors/pennychat.py

```diff
@@ -356,21 +353,15 @@ def time_select(self, event):
     def user_select(self, event):
         users = event['actions'][0]['selected_users']
         penny_chat = PennyChat.objects.get(view=event['view']['id'])
-        invitees = ''
-        for invitee in users:
-            invitees = invitees + ',' + invitee if len(invitees) > 0 else invitee
-        penny_chat.invitees = invitees
+        penny_chat.invitees = ','.join(users)
         penny_chat.save()
 
     @is_block_interaction_event
     @is_action_id('penny_chat_channel_select')
     def channel_select(self, event):
         selected_channels = event['actions'][0]['selected_channels']
         penny_chat = PennyChat.objects.get(view=event['view']['id'])
-        channels = ''
-        for channel in selected_channels:
-            channels = channel + ',' + channel if len(channels) > 0 else channel
```

Comment:
here's where the bug was - shoulda been `channels = channels + ',' + channel ...` note the plural